src/sass/_Fabric.Typography.Variables.scss

```diff
@@ -5,11 +5,18 @@
 // --------------------------------------------------
 // Fabric Core Typography variables
 
+$ms-font-system-base: 'Segoe UI', -apple-system, BlinkMacSystemFont, 'Roboto', 'Helvetica Neue', sans-serif;
+$ms-font-family:  'Segoe UI WestEuropean', $ms-font-system-base;
```

Comment:
Super small, but it feels like this should be more descriptive, e.g. `$ms-font-family-base` or something like that.